libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/recenter/RecenterButtonBehaviour.kt

```diff
@@ -3,19 +3,18 @@ package com.mapbox.navigation.dropin.component.recenter
 import com.mapbox.maps.CameraOptions
 import com.mapbox.navigation.dropin.component.camera.DropInCameraState
 import com.mapbox.navigation.dropin.component.camera.DropInCameraState.CameraUpdateEvent.EaseTo
-import com.mapbox.navigation.dropin.component.location.LocationBehavior
+import com.mapbox.navigation.dropin.component.location.LocationViewModel
 import com.mapbox.navigation.dropin.lifecycle.UIComponent
-import com.mapbox.navigation.utils.internal.toPoint
 
 internal class RecenterButtonBehaviour(
```

Comment:
Let's stick with the American spelling of **behavior**.

Reply:
Yeah we're actually deleting Behavior in favor of ViewModel. This pull request is migrating some of the pieces

Reply:
Addressed here https://github.com/mapbox/mapbox-navigation-android/pull/5534